sdk/ios/ConfigsumTests/contextWithComplexMetadata.json

```diff
@@ -18,7 +18,7 @@
         },
         "location" : {
             "locale" : "en-US",
-            "timezoneOffset" : 7200
+            "timezoneOffset" : 3600
```

Comment:
Any reason these were changed? :)

Reply:
code for calculating timezoneOffset was added